README.md

```diff
@@ -17,16 +17,16 @@ F´ (F Prime) is a component-driven framework that enables rapid development and
 - A growing collection of ready-to-use components
 - Testing tools for testing flight software at the unit and integration levels.
 
-Learn more about [F´ key features](https://nasa.github.io/fprime/features.html).
+Learn more about [F´ key features]([https://(https://fprime.jpl.nasa.gov/latest/)).
```

Comment:
it seems to me like there's a typo in this line: an extra opening square bracket.

Reply:
Thank you for catching that.
 - Removed extra bracket on hyperlink
 - Updated website to current one: 
**Please Visit the F´ Website:** [https://fprime.jpl.gov/latest](https://fprime.jpl.nasa.gov/latest/)